app/assets/javascripts/controllers/ownership/ownership_form_controller.js

```diff
@@ -0,0 +1,77 @@
+ManageIQ.angular.app.controller('ownershipFormController', ['$http', '$scope', 'objectIds', 'miqService', function($http, $scope, objectIds, miqService) {
+  var init = function() {
+    $scope.ownershipModel = {
+      user: '',
+      group: ''
+    };
+    $scope.afterGet  = false;
+    $scope.newRecord = false;
+    $scope.modelCopy = angular.copy( $scope.ownershipModel );
+    $scope.model     = "ownershipModel";
+    $scope.objectIds = objectIds;
+    $scope.saveable = miqService.saveable;
+
+    ManageIQ.angular.scope = $scope;
+
+    miqService.sparkleOn();
+    $http.get('ownership_form_fields/' + objectIds).success(function(data) {
+      $scope.ownershipModel.user = data.user;
+      $scope.ownershipModel.group = data.group;
+      $scope.afterGet = true;
+      $scope.modelCopy = angular.copy( $scope.ownershipModel );
+      miqService.sparkleOff();
+    });
+  };
+
+  $scope.canValidateBasicInfo = function () {
+    if ($scope.isBasicInfoValid())
+      return true;
+    else
+      return false;
+  };
+
+  $scope.isBasicInfoValid = function() {
+    if(( $scope.angularForm.user && !$scope.angularForm.user.$valid) ||
+      ($scope.angularForm.group && !$scope.angularForm.group.$valid))
+      return false;
+    else
+      return true;
+  };
+
+
+  var ownershipEditButtonClicked = function(buttonName, serializeFields) {
+    miqService.sparkleOn();
+    var url = 'ownership_update/' + objectIds[0] + '?button=' + buttonName;
```

Comment:
this feels weird, why just that one id, if we're supporting more..? I do notice that i'ts only used for cancel like like this .. but then why pass the ID in the url at all?